packages/create-cloudflare/templates/hono/c3.ts

```diff
@@ -1,24 +1,77 @@
 import { logRaw } from "@cloudflare/cli";
+import { brandColor } from "@cloudflare/cli/colors";
+import { spinner } from "@cloudflare/cli/interactive";
 import { runFrameworkGenerator } from "frameworks/index";
+import { loadTemplateSnippets, transformFile } from "helpers/codemod";
+import { detectPackageManager } from "helpers/packageManagers";
 import type { TemplateConfig } from "../../src/templates";
+import type * as recast from "recast";
 import type { C3Context } from "types";
 
 const generate = async (ctx: C3Context) => {
+	const { name: pm } = detectPackageManager();
+
 	await runFrameworkGenerator(ctx, [
 		ctx.project.name,
 		"--template",
 		"cloudflare-workers",
+		"--install",
+		"--pm",
+		pm,
 	]);
 
 	logRaw(""); // newline
 };
 
+const configure = async (ctx: C3Context) => {
+	const indexFile = "src/index.ts";
+
+	const s = spinner();
+	s.start(`Updating \`${indexFile}\``);
+
+	const snippets = loadTemplateSnippets(ctx);
+
+	transformFile(indexFile, {
+		// Insert the env declaration after the last import (but before the rest of the body)
+		visitProgram: function (n) {
+			const lastImportIndex = n.node.body.findLastIndex(
+				(t) => t.type === "ImportDeclaration"
+			);
+			const lastImport = n.get("body", lastImportIndex);
+			lastImport.insertAfter(...snippets.bindingsTypeTs);
+
+			return this.traverse(n);
+		},
+		visitVariableDeclarator(n) {
+			if (n.node.id.type === "Identifier" && n.node.id.name === "app") {
+				n.node.init = snippets
+					.appDeclarationTs[0] as recast.types.namedTypes.NewExpression;
+
+				return false;
+			}
+		},
+	});
+
+	s.stop(`${brandColor("updated")} \`${indexFile}\``);
```

Comment:
```suggestion
	s.stop(`${brandColor("updated")} \`${dim(indexFile)}\``);
```

Usually the updated file name is dimmed right?
![Screenshot 2024-04-22 at 10 07 28](https://github.com/cloudflare/workers-sdk/assets/61631103/27ab8f7e-b843-445f-8a92-d79a771d523d)

(only in Qwik I see an exception to this and I'd imagine we'd want to amend that too? 🤔)